assets/pages/subscriptions-landing/components/featuredProducts.jsx

```diff
@@ -95,6 +96,22 @@ const paperImage = (
     fallbackImgType="png"
   />);
 
+const getDigitalPackSubHeadingAndBody = (countryGroupId: CountryGroupId) => {
+  if (flashSaleIsActive('DigitalPack')) {
+    return {
+      subheadingText: 'Save 50% for three months',
+      bodyText: `Read the Guardian ad-free on all devices, including the Premium App and Daily Edition iPad app. ${discountedDisplayPrice('DigitalPack', countryGroupId)} for your first three months.`,
```

Comment:
I wish we would say '... ad-free on desktop and mobile devices ...' here. Not a problem for this PR but something to raise elsewhere I guess.